t/admin/balancer.t

```diff
@@ -70,9 +70,9 @@ __DATA__
             local up_conf = {
                 type = "roundrobin",
                 nodes = {
-                    {host = "39.97.63.215", port = 80, weight = 1},
-                    {host = "39.97.63.216", port = 81, weight = 1},
-                    {host = "39.97.63.217", port = 82, weight = 1},
+                    {host = "39.97.63.215", port = 80, weight = 1, priority = 0},
```

Comment:
the default value of `priority` is `0`, so we can keep it not change, all right?

Reply:
Yes.